.github/workflows/pre-release.yml

```diff
@@ -18,9 +18,22 @@ jobs:
 
       - uses: actions/checkout@v3
       - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
+        run: mvn --batch-mode --update-snapshots install
       - run: mkdir staging && cp drools-lsp-server/target/drools-lsp-server-jar-with-dependencies.jar staging
 
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Package VS Code Extension
+        working-directory: ./client
+        run: |
+          npm install
+          npm install -g vsce
+          vsce package
+          cp *.vsix ../staging/
```

Comment:
You can move this code to a script inside your `package.json` so that it can be run both by CI a locally.
Example on kie-tools [1]. Anyway, this can be done on [2].

Also, `vsce` is deprecated in favor of `@vscode/vsce`. See [3].
I suggest specifying the version of the dependencies you install to avoid surprises.
Example: `npm install -g @vscode/vsce@2.19.0`

[1] https://github.com/kiegroup/kie-tools/blob/main/packages/vscode-extension-serverless-workflow-editor/package.json#L23
[2] https://issues.redhat.com/browse/DROOLS-7404
[3] https://www.npmjs.com/package/vsce

Reply:
Thanks, done.